responsible.php

```diff
@@ -119,7 +119,7 @@ function responsible_menu() {
 	 * @return string
 	 */
 	function bookmarklet() {
-		return apply_filters( 'responsible_bookmarklet', "javascript:void((function(d){if(self!=top||d.getElementById('toolbar')&&d.getElementById('toolbar').getAttribute('data-resizer'))return false;d.write('<!DOCTYPE HTML><html style=&quot;opacity:0;&quot;><head><meta charset=&quot;utf-8&quot;/></head><body><a data-viewport=&quot;320x480&quot; data-icon=&quot;mobile&quot;>Mobile (e.g. Apple iPhone)</a><a data-viewport=&quot;320x568&quot; data-icon=&quot;mobile&quot; data-version=&quot;5&quot;>Apple iPhone 5</a><a data-viewport=&quot;600x800&quot; data-icon=&quot;small-tablet&quot;>Small Tablet</a><a data-viewport=&quot;768x1024&quot; data-icon=&quot;tablet&quot;>Tablet (e.g. Apple iPad 2-3rd, mini)</a><a data-viewport=&quot;1280x800&quot; data-icon=&quot;notebook&quot;>Widescreen</a><a data-viewport=&quot;1920×1080&quot; data-icon=&quot;tv&quot;>HDTV 1080p</a><script src=&quot;http://lab.maltewassermann.com/viewport-resizer/resizer.min.js&quot;></script></body></html>')})(document));" );
+		return apply_filters( 'responsible_bookmarklet', "javascript:void((function(d){if(self!=top||d.getElementById('toolbar')&&d.getElementById('toolbar').getAttribute('data-resizer'))return false;d.write('<!DOCTYPE HTML><html style=&quot;opacity:0;&quot;><head><meta charset=&quot;utf-8&quot;/></head><body><a data-viewport=&quot;320x480&quot; data-icon=&quot;mobile&quot;>Mobile (e.g. Apple iPhone)</a><a data-viewport=&quot;320x568&quot; data-icon=&quot;mobile&quot; data-version=&quot;5&quot;>Apple iPhone 5</a><a data-viewport=&quot;600x800&quot; data-icon=&quot;small-tablet&quot;>Small Tablet</a><a data-viewport=&quot;768x1024&quot; data-icon=&quot;tablet&quot;>Tablet (e.g. Apple iPad 2-3rd, mini)</a><a data-viewport=&quot;1280x800&quot; data-icon=&quot;notebook&quot;>Widescreen</a><a data-viewport=&quot;1920×1080&quot; data-icon=&quot;tv&quot;>HDTV 1080p</a><script src=&quot;//lab.maltewassermann.com/viewport-resizer/resizer.min.js&quot;></script></body></html>')})(document));" );
```

Comment:
I'd stick to http always rather than using protocol relative URLs

Reply:
What is your thinking here Tom? I had an issue from ages ago that I only just spotted saying that it does not work on `https` sites. Would be good to fix that I think.

Reply:
@tomjn can you explain why? It would ideally work on both http and https sites. I guess I can use WordPress to work that out for me.